modules/images/regenerate-existing-images/load.php

```diff
@@ -7,3 +7,79 @@
  * @since n.e.x.t
  * @package performance-lab
  */
+
+/**
+ * Register the background job taxonomy.
+ *
+ * Intentionally on lower priority, so that other post types can be
+ * registered successfully and taxonomy can be used for all the post types.
+ *
+ * This taxonomy will be used as a queue system where each term represents a job in the queue.
+ *
+ * @return void
```

Comment:
`@return` docs should be omitted when it's just `void`, see WordPress core documentation standards.

Instead, we need to add a `@since n.e.x.t` tag here, as for every function/class/method etc.